setup.py

```diff
@@ -216,9 +222,12 @@ def run(self):
                     "libcudnn_heuristic.so.9",
                     "libcudnn_ops.so.9",
                     "libnvJitLink.so.12",
+                    "libnvrtc.so.11",
```

Comment:
For some reason, libnvrtc.so.11 and libnvrtc-builtins.so.11 are still in package. Size is about 20MB per whl file.

Reply:
Yeah, we need to solve this problem.